frontend/src/components/common/form/ContactInput.tsx

```diff
@@ -99,13 +99,38 @@ export const ContactInput: React.FC<ContactInputProps> = ({
 
 const StyledDiv = styled.div`
   border-radius: 0.3rem;
-  padding: 12px 6px 6px 12px;
+  padding: 6px;
   min-height: 2.5em;
   background-image: none;
-  background-color: ${props => props.theme.css.filterBackgroundColor};
+  color: #495057;
```

Comment:
we should use the colors in the theme. If there is no color that matches it please create one there. If there is only a small difference between the existing colors and the one specified we should chat about it. Otherwise we will get 100 shades of blue

Reply:
Will look for the color in theme and try to use it.

Reply:
removed and used from themes